internal/migrations/statik.go

```diff
@@ -0,0 +1,14 @@
+// Code generated by statik. DO NOT EDIT.
```

Comment:
I think it would be good to:

* Move this file to the `internal/integration/postgresql` directory
* Add it to `.gitignore`
* Make sure it is generated on `make` (like with the other migrations)